frontend/src/components/experimentRuns/ExperimentRunsDetachedFilters.ts

```diff
@@ -0,0 +1,14 @@
+import { FilterDescription } from "../../models/paged_items";
+import { FILTER_TYPE } from '../../constants'
+
+export const EXPERIMENT_RUNS_PLATFORM_NAME_FILTER: FilterDescription = {
```

Comment:
This is unmerged code from your other PR ...

Reply:
closing this pr because the actualy code needed to test the validation email has been brought into pr https://github.com/c3g/freezeman/pull/831